tox.ini

```diff
@@ -1,9 +1,10 @@
 [tox]
 envlist =
     py39-django{42}
-    py310-django{42,main}
-    py311-django{42,main}
-    py312-django{42,main}
+    py310-django{42,52}
+    py311-django{42,52}
+    py312-django{42,52,main}
+    py313-django{42,52,main}
```

Comment:
Django 5.1 EOL is at the end of this year, so we might want to still run tests with it.